### ISSUE TITLE
terranix.org seems to be down

### ISSUE THREAD

**Issue by tuxiqae** [terranix/terranix.github.io#12]
https://terranix.github.io redirects to https://terranix.org which is down, seems to be something with the DNS record (at the very least) since the host cannot be resolved.

**Comment by mrVanDalo** [comment]
Thanks for the info. It works for me, but DNS is sometimes strange.

It should be the opposite, https://terranix.org should redirect to terranix.github.io.
I'm on it.

**Comment by mrVanDalo** [comment]
I enabled accidentally DNSSEC on namecheap.com the other day, and this could have created the problem. I disabled DNSSEC for now. Hope it will work soon. @tuxiqae could you ping me when soonish, if it works?

**Comment by tuxiqae** [comment]
Done!
Both seem to work (even though https://terranix.github.io still seem to redirect to https://terranix.org, but my understanding of CNAMEs might be wrong)

**Comment by mrVanDalo** [comment]
Strange, I don't get a CNAME on `dig terranix.github.io`  or `dig CNAME terranix.github.io`
But my DNS skills are not the best.

**Comment by tuxiqae** [comment]
Yeah, the CNAME is on the other end (https://terranix.org) but when I browse to https://terranix.github.io  it redirects to https://terranix.org

**Comment by mrVanDalo** [comment]
I see, if you `curl https://terranix.github.io` you'll get a redirect (301), because the `Host` is not correctly set. But this seems how github pages handles this problem.
